### DISCUSSION TITLE
Is there any case that z3 reports 'unsat' when logic is insufficient?

### DISCUSSION THREAD

**Discussion by kangwoosukeq** [Z3Prover/z3#6282] [Q&A] [answered]
According to #3916, there is a case that z3 reports 'sat' when given logic doesn't cover some components of the expression.

Then, I want to ask if there is any possible case that z3 reports 'unsat' when a part of the expression is expressed uninterpreted because of its logic.

**Comment by NikolajBjorner** [comment, marked as answer]
no
